### ISSUE TITLE
Russian Language missed after update to 1.0.1.0 

### ISSUE THREAD

**Issue by ARGAMX** [Keriew/augustus#449]
Hello.

I download Julius and Augustus for my Caesar III. My Caesar III is Russian version.
I have run Julius and Augustus but got message about patch 1.0.1.0 needed.
I have downloaded Russian patch, extracted and overwrite to game folder.
This didn't change anything - got the same error about patch 1.0.1.0 needed.

Ok.
I have downloaded English patch, extracted and overwrite to game folder.
After that 1.0.1.0 error are gone.
But game now in English.

I have extracted Russian patch and overwrite to game folder again.
Game still in English.

In game settings there is no option to select language (In Julius and in Augustus).
https://i.postimg.cc/0PY3sB5y/Screenshot-2021-07-05-at-21-04-54.png

Looks like a bug.
Thanks!

**Comment by MSTVD** [comment]
What version are you playing on? This does not look like 1C

**Comment by ARGAMX** [comment]
> What version are you playing on? This does not look like 1C

But how I can Identify it? This is the version I was playing when I was young.

BTW I have found how the problem can be fixed:

Firstly I have created folder 'rus' under the game folder.
After that I have downloaded Russian patch and extracted it to that 'rus' folder.
Now in Language selection there is 'rus' language available.

But seems that path is bit complicated and not obvious.
